docs/explanations/principles.md

```diff
@@ -0,0 +1,106 @@
+
+
+# Gutenberg Principles
+
+This document has been written so that those contributing to the Gutenberg project can have some understanding of what guides decisions around what is included in the project (or not).
+
+An important thing to keep at top of mind when reviewing these principles is that they are not isolated from each other. There is a cohesiveness and complementary nature between them that results in a synergistic impact and direction.
+
+When applied, sometimes these principles build on each other much like the idea of constructing a house with a foundation, walls and roof. However, other times the application of the principles results in something that is more similar to a tapestry where threads of each principle are embedded together and evident in the finished feature or design of the project.
+
+
+## Aim for delightful, consistent, accessible user interfaces for empowering content creation and expression.
+
+_“Optimize for the user” (Matias - [Gutenberg or the Ship of Theseus](https://matiasventura.com/post/gutenberg-or-the-ship-of-theseus/))_
+
+Each of the words in this principle are deliberate:
+
+
+### Delightful
+
+The user interfaces should “get out of the way” and support the user in creating their content. They should lead to experiences where the user (in the vernacular of Marie Kondo) _finds delight_ with each new thing they discover they can do or accomplish.
+
+
+### Consistent
```

Comment:
This is not a principle I'd be comfortable stating as a primary guiding force because it's a bit more nuanced. The way I tend to describe this is more along the lines of "Providing uniformity within diversity, as learning the ingredients of the interface happens once but scales to hundreds of blocks."

This is an important design tension.

Reply:
I like the concept of "providing uniformity within diversity" but I think there's still the importance here of capturing the idea of not just the _block interfaces_ but also the interfaces within the editor? Would you say this statement is a principle that also applies to the editor interfaces?

Also, is this something that you believe should be explicitly called out as it's _own_ principle, or should it be included somehow as a part of [the statement commented on here](https://github.com/WordPress/gutenberg/pull/30963#discussion_r620302895)?

Reply:
See d6db504.